search/search_service/api/swagger_doc/table/search_table_filter.yml

```diff
@@ -3,36 +3,32 @@ This is used by the frontend API to search table information.
 ---
 tags:
   - 'search_table'
-paths:
-  /search_table:
-    post:
-      summary: This is used by the frontend API to search table information.
-      requestBody:
-        description: The json data passed from the frontend API to execute a search.
-        required: true
-        content:
-          application/json:
-            schema:
-              type: object
-              properties:
-                index:
-                  type: string
-                page_index:
-                  type: integer
-                query_term:
-                  type: string
-                search_request:
-                  type: object
-      responses:
-        200:
-          description: table result information with query string
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/SearchTableResults'
-        500:
-          description: Exception encountered while searching
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/ErrorResponse'
+# Edited: Removed paths, post and summary
```

Comment:
can you remove this comment? This should just live in commit history

Reply:
Done 👍🏻